docs/data/data-grid/editing/ValidateServerNameGrid.js

```diff
@@ -23,13 +23,10 @@ function validateName(username) {
   const existingUsers = rows.map((row) => row.name.toLowerCase());
 
   return new Promise((resolve) => {
```

Comment:
No idea why this formatting change, if you run `yarn prettier` does it revert it?

Reply:
running `yarn prettier` modifies some files. let see if it fixes the CI

Reply:
Yeah, similarly with `pnpm docs:typescript:formatted` — it was picking up files I hadn't touched on this PR, so I just avoided running those.

Reply:
Could have been some prettier settings 🤔 Should be fine now I think

Reply:
Sweet! Thanks, y'all! Would appreciate a review now :)